nimbus-shared/generate/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "generate"
+version = "0.1.0"
+authors = []
+# This is just for circle, so does not need to be published
+publish = false
+
+[dependencies]
+schemars = "0.8.0"
+serde_json = "1.0"
```

Comment:
This dependency version of `"1.0"` doesn't exactly match up with the base Cargo.toml dependency on serde listed as `"1"`.  Not sure that would cause any major problems or not, but you may not quite resolve the version you expect in one or the other place.  If you make the changes suggested by @rfk for the structure, then this shouldn't be a problem anymore.